### ISSUE TITLE
Big datasets

### ISSUE THREAD

**Issue by mommi84** [AKSW/LIMES-dev#4]
Can the current workflow deal with big datasets (i.e., when it's impossible to store them in-memory)?

**Comment by ngonga** [comment]
Yes. See memory management package. The mapping class needs to be updated though. We need a file mapping that supports writing mappings to the hard drive.

**Comment by mommi84** [comment]
Okay. I would keep this issue open until the new `Mapping` class is updated.

**Comment by Kleanthi** [comment]
How is this thing going?

**Comment by dobraczka** [comment]
Kevin and I are currently working on porting HR3 to either Flink or Spark. Though this task is certainly smaller than the scope of the original question it might be reasonable to aim for such frameworks rather than having a new ```Mapping``` class, i.e. having a LIMES-Flink oder LIMES-Spark implementation, that can be run in a cluster.

**Comment by Kleanthi** [comment]
I like the LIMES-Spark idea.

**Comment by kvndrsslr** [comment]
I did some research on this lately and it seems like [Apache BEAM](https://beam.apache.org) is what we'd want for a complete LIMES port to big data technology.
Will be considered as part of the upcoming rewrite.
